src/main/resources/package.json

```diff
@@ -1,24 +1,158 @@
 [
   {
     "id": 1,
-    "name": "TestName",
-    "testType": "A",
+    "name": "Effektive Reichweite gerade",
+    "testType": "SingleTagMultipleReads",
     "preParameters": [
-      "Please insert some params"
+      "AmountOfReads",
+      "OrientationBetweenTagAndAntenna"
     ],
     "postParameters": [
-      "Please insert some post params"
+      "DistanceBetweenTagAndAntenna"
     ]
   },
   {
     "id": 2,
-    "name": "TestName",
-    "testType": "A",
+    "name": "Lesegeschwindigkeit Bulk Reading",
+    "testType": "MultipleTagsMultipleReads",
     "preParameters": [
-      "Please insert some params"
+      "AmountOfReads",
+      "DistanceBetweenTagsAndAntenna",
```

Comment:
Jeder Text hier sollte wirklich leserlich sein, also nicht camelcase verwenden und allenfals auch besser in deutsch

Reply:
👍